tools/pepsi/src/aliveness.rs

```diff
@@ -26,15 +26,25 @@ pub async fn aliveness(arguments: Arguments) -> anyhow::Result<()> {
         Arguments::Disable { naos } => (false, naos),
     };
 
-    let tasks = naos.into_iter().map(|nao_address| async move {
-        let nao = Nao::new(nao_address.ip);
-        nao.set_aliveness(enable)
-            .await
-            .with_context(|| format!("Failed to set aliveness on {nao_address}"))
+    let multi_progress = ProgressIndicator::new();
+
+    let tasks = naos.into_iter().map(|nao_address| {
+        let multi_progress = multi_progress.clone();
```

Comment:
@schmidma you once showed me a trick on how to clone variables to be used in threads and async tasks. Do you remember what that was? But this line looks quite good IMHO. 🤔

Reply:
I think there is no 'trick'. You have to clone the variable. IIRC the only remark I made on your code in the past was to clone the variable in an extra scope, naming it  equally and thus shadow the previous variable. This removes these odd `variable2` or `variable_cloned` naming.
IMHO, this looks fine.

Reply:
Thanks for your input. Resolving.